src/controller/python/chip/native/__init__.py

```diff
@@ -93,7 +138,55 @@ def Init(bluetoothAdapter: int = None):
     params = CommonStackParams.build(params)
 
     _GetLibraryHandle(False).pychip_CommonStackInit(ctypes.c_char_p(params))
+    Api = _BuildApi()
+    _nativeLibraryInitialized = True
 
 
-def GetLibraryHandle():
-    return _GetLibraryHandle(True)
+# Note: Below is the experimental API for calling functions and convert ChipError to throwing exceptions fluently.
+
+def _GetNativeFunc(func: str):
+    return getattr(_GetLibraryHandle(False), func)
+
+
+def _DmlibFunc(func: str, ret_type: type, arg_types: List[type]) -> Callable:
+    ''' Wraps the func from Matter dynamic library into Python function.
+
+    If ret_type is ChipStackError, calling it will raise an exception if the underlying function returns an non-zero error code.
```

Comment:
I'm a little confused here - none of the C binding APIs return `ChipStackError` right? Or is this referring to a different function?

Reply:
This function no longer exists